test/unittest/compiler/unittest_tflite_export.cpp

```diff
@@ -0,0 +1,396 @@
+// SPDX-License-Identifier: Apache-2.0
+/**
+ * Copyright (C) 2023 Donghak Park <donghak.park@samsung.com>
+ *
+ * @file unittest_tflite_export.cpp
+ * @date 23 November 2023
+ * @brief export test
+ * @see	https://github.com/nnstreamer/nntrainer
+ * @author Donghak Park <donghak.park@samsung.com>
+ * @bug No known bugs except for NYI items
+ */
+
+#include <functional>
+#include <gtest/gtest.h>
+#include <memory>
+#include <sstream>
+#include <vector>
+
+#include <app_context.h>
+#include <flatten_realizer.h>
+#include <layer.h>
+#include <model.h>
+#include <node_exporter.h>
+#include <optimizer.h>
+#include <realizer.h>
+
+#include <nntrainer_test_util.h>
+
+#ifdef ENABLE_TFLITE_INTERPRETER
+#include <tensorflow/lite/interpreter.h>
+#include <tensorflow/lite/kernels/register.h>
+#include <tensorflow/lite/model.h>
+#include <tflite_interpreter.h>
+#endif
+
+using LayerRepresentation = std::pair<std::string, std::vector<std::string>>;
+using LayerHandle = std::shared_ptr<ml::train::Layer>;
+using ModelHandle = std::unique_ptr<ml::train::Model>;
+using ml::train::createLayer;
+
+std::vector<float> out;
+std::vector<float> ans;
+std::vector<float *> in_f;
+std::vector<float *> l_f;
+
+/**
+ * @brief make "key=value" from key and value
+ *
+ * @tparam T type of a value
```

Comment:
```suggestion
 * @param T type of a value
```
typo?

Reply:
It mean template parameters. in doxygen they guide like below
```Use @tparam for template parameters. Start the explanation with upper case and end it with dot.```

Reply:
didn't know that. thanks for letting me know :)